Multisig/UI/App/UIKit/Transaction/TransactionDetailsViewController/TransactionDetailCellBuilder.swift

```diff
@@ -371,31 +371,38 @@ class TransactionDetailCellBuilder {
         }
 
         func buildAdvanced() {
-            let nonce: String?
-            let operation: String? = tx.txData?.operation.string
-            let hash: String? = tx.txHash?.description
-            let safeTxHash: String?
-
-            if case SCG.TransactionDetails.DetailedExecutionInfo.multisig(let multisigTx)? =
-                tx.detailedExecutionInfo {
-                nonce = multisigTx.nonce.description
-                safeTxHash = multisigTx.safeTxHash.description
-            } else {
-                nonce = nil
-                safeTxHash = nil
-            }
+            switch tx.txInfo {
+            case .transfer(let transferTx):
+                guard transferTx.direction != .incoming else { return }
```

Comment:
Are you sure this is what the ticket is about? Now the incoming transaction's transactionHash will not be shown at all, right?

Reply:
This is what is meant in ticket description right ?
https://app.zenhub.com/workspaces/safe-multisig---mobile-5c1b98250e13551e8aae81eb/issues/gnosis/safe-ios/578
and transactionHash is accessible by "View on Eathscan" link right?

Reply:
aah! I thought that we need to display tx hash in the details